packit_service/worker/handlers/copr.py

```diff
@@ -479,3 +496,156 @@ def handle_testing_farm(self):
                         "build_id": self.build.id,
                     },
                 ).apply_async()
+
+    def handle_scan(self):
+        """
+        If configured, try to find a job that can provide the base SRPM,
+        download both SRPM and base SRPM and trigger the scan in OpenScanHub.
+        """
+        if (
+            self.db_project_event.type != ProjectEventModelType.pull_request
+            or self.build.target != "fedora-rawhide-x86_64"
+            or not self.job_config.osh_diff_scan_after_copr_build
+        ):
+            return
+
+        base_build_job = self.find_base_build_job()
+        if not base_build_job:
+            logger.debug("No base build job needed for diff scan found in the config.")
+            return
+
+        logger.info("Preparing to trigger scan in OpenScanHub...")
+
+        base_srpm_build = self.get_base_srpm_build(base_build_job)
+
+        if not base_srpm_build:
+            return
+
+        srpm_model = self.build.get_srpm_build()
+
+        directory = tempfile.mkdtemp()
+        paths = self.download_srpms(directory, base_srpm_build, srpm_model)
+        if not paths:
+            return
+
+        output = self.copr_build_helper.api.run_osh_build(
+            srpm_path=paths[1],
+            base_srpm=paths[0],
+            comment="Submitted via Packit Service.",
```

Comment:
Please leave url to the task dashboard here.

Reply:
good point, added